H_ecto_models.md

```diff
@@ -243,13 +237,17 @@ Changesets define a pipeline of transformations our data needs to undergo before
 Let's take a closer look at our default changeset.
 
 ```elixir
-def changeset(model, params \\ :empty) do
-  model
-  |> cast(params, @required_fields, @optional_fields)
-end
+  def changeset(struct, params \\ %{}) do
+    struct
+    |> cast(params, [:name, :email, :bio, :number_of_pets])
+    |> validate_required([:name, :email, :bio, :number_of_pets])
+  end
 ```
-
-At this point, we only have one transformation in our pipeline. This `cast/4` function's main job is to separate required fields from optional ones. We define the fields for each category in the module attributes `@required_fields` and `@optional_fields`. By default all of the fields are required.
+Right now, we have two simple pieces to this model.
```

Comment:
I think we should keep this as "transformations in our pipeline" or "functions in our pipeline" instead of "simple pieces"